docs/sources/shared/configuration.md

```diff
@@ -491,23 +491,22 @@ pattern_ingester:
   # CLI flag: -pattern-ingester.max-eviction-ratio
   [max_eviction_ratio: <float> | default = 0.25]
 
-  # Configures the metric aggregation and storage behavior of the pattern
-  # ingester.
-  metric_aggregation:
-    # How often to downsample metrics from raw push observations.
-    # CLI flag: -pattern-ingester.metric-aggregation.downsample-period
-    [downsample_period: <duration> | default = 10s]
+  # Configures the aggregation and storage behavior of the pattern ingester.
+  aggregation:
```

Comment:
i wonder if rather than making this generic, we should have separate configs for metric_aggregation and pattern persistence.

I think we are overloading concepts here in a way that is a bit confusing and potentially harder to optimize in the future as we replace our storage (a different label identifier for these types would be important for handling them differently if we choose to in the future)